src/System.Net.Http.WinHttpHandler/tests/UnitTests/ClientCertificateHelper.cs

```diff
@@ -0,0 +1,271 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Security.Cryptography.X509Certificates;
+
+using Xunit;
+
+namespace System.Net.Http.WinHttpHandlerUnitTests
+{
+    public class ClientCertificateHelper
+    {
+        private readonly X509Certificate2 _cert_KeyUsageIncludesDigitalSignature_EKUIncludesClientAuth_PrivateKey =
+            new X509Certificate2(
+                Convert.FromBase64String(
+                  @"MIIKTgIBAzCCCgoGCSqGSIb3DQEHAaCCCfsEggn3MIIJ8zCCBgwGCSqGSIb3DQEHAaCCBf0EggX5
```

Comment:
PFX files are binary and shouldn't be checked into the source tree.  But you can take all the bytes from PFX file, Base64encode them as I did for some of the certificates in the ClientCertificateHelper class.

Some of the certs in this file have private keys.  But they are BRAND NEW certificates I generated with ficticious X509Certificate.SubjectName (CN=user@example.com).  The public/private key pair is newly generated and has no PII.